src/code_e20eb0.c

```diff
@@ -1,6 +1,20 @@
 #include "common.h"
 
-INCLUDE_ASM(code_e20eb0, func_802B7000);
+// TODO: Figure out what this function is doing
+s32 func_802B7000(f32 angle) {
+    s32 angleDiff;
+
+    if (270.0f <= angle) {
```

Comment:
can the `angle` be on the left side of these conditionals? I hope so...

Reply:
I tried and it cannot

Reply:
it can be; ``angle >= 270.0f``in my matching copy locally. Same pattern for the rest

Reply:
Ah, I was doing `angle > 270.f` like a dummy. Thank you. Sending up the revised version...